plugins/broker-redis/src/static/generateRedisClientOptions.ts

```diff
@@ -0,0 +1,33 @@
+import { ConfigService } from "@nestjs/config";
+import { RedisOptions, Transport } from "@nestjs/microservices";
+
+export const generateRedisClientOptions = (
+    configService: ConfigService
+): RedisOptions => {
+  const redisEnableTLS = configService.get("REDIS_BROKER_ENABLE_TLS") === "true";
+  const redisUrl = configService.get("REDIS_BROKER_URL");
+  const redisRetryAttempts = configService.get("REDIS_BROKER_RETRY_ATTEMPTS");
+  const redisRetryDelay = configService.get("REDIS_BROKER_RETRY_DELAY");
```

Comment:
If we already have `constats.ts` file, we can add these variables to this file and use it here

Reply:
@abrl91, I don't really understand what you're saying here. Add which variables to which file?

Reply:
for example, instead of this:
```
const redisEnableTLS = configService.get("REDIS_BROKER_ENABLE_TLS") === "true";
```

doing this
```
import {REDIS_BROKER_ENABLE_TLS} from "constants"
const redisEnableTLS = configService.get(REDIS_BROKER_ENABLE_TLS) === "true";
```

Because we already generate a constants file, lets use it also for these variables